apps/web/src/hooks/useSearchHotkey.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useRef } from 'react';
+
+export function useSearchHotkey(callback: () => void) {
+  const callbackRef = useRef(callback);
+  callbackRef.current = callback;
+
+  useEffect(() => {
+    function handlePress(event: KeyboardEvent) {
+      if (event.key === 'k' && (event.metaKey || event.ctrlKey)) {
+        // Avoid trying to save page as HTML
+        event.preventDefault();
```

Comment:
i see you copy pastad lol, do we need this for cmd k? maybe the useMultiKeyDown will work?

Reply:
Removed the `event.preventDefault` 🤦

Tried with `useMultiKeyDown` but it doesnt go smooth as expected since we wanna support this both commands just like you did on editor shortcut save

1. Control + k
2. Meta + k


Tried with this snippets and it seems doesnt work well. Might need to add support multile shortcut in the `useMultiKeyDown` hooks

```tsx
  useMultiKeyDown(['Meta', 'k'], handleShowSearch);
  useMultiKeyDown(['Control', 'k'], handleShowSearch);
```

Reply:
gotcha, thanks @jakzaizzat  - I think handling Meta and Control keys will be a common use case for keyboard shortcuts so maybe it should be baked into the `useMultiKeyDown` hook. I don't think it blocks this PR though